tests/tests/features/cronjobs.yaml

```diff
@@ -62,6 +62,24 @@
   tasks:
   - include: ../../checks/check-url-content.yaml
 
+# This is where we need to modify the lagoon versions and do another deploy
+- name: "{{ testname }} - change .lagoon.yml to remove cron, git push"
+  hosts: localhost
+  serial: 1
+  vars:
+    git_files: "features/"
+  tasks:
+  - include: ../../tasks/git-update-lagoon-commit-push.yaml
+
```

Comment:
after changing the code, you need to trigger a deployment first via `../../tasks/api/deploy-no-sha.yaml`, see https://github.com/amazeeio/lagoon/pull/1178/files#diff-22bec30a3d720f348368edc7531a31b3R11-R18